satellite/console/service.go

```diff
@@ -119,22 +119,41 @@ func (s *Service) CreateUser(ctx context.Context, user CreateUser, tokenSecret R
 		return nil, errs.New(internalErrMsg)
 	}
 
-	u, err = s.store.Users().Insert(ctx, &User{
-		Email:        user.Email,
-		FullName:     user.FullName,
-		ShortName:    user.ShortName,
-		PasswordHash: hash,
-	})
+	// store data
+	tx, err := s.store.BeginTx(ctx)
 	if err != nil {
-		return nil, errs.New(internalErrMsg)
+		return nil, err
```

Comment:
Is there a reason why we aren't wrapping this error?

Reply:
that's not wrapping, but replacing error contents completely with user friendly, but non-descriptive error. I am not fan of this approach, but if needed it can be replaced in the future after some testing period, because it's much easier to identify errors/bugs with real error msgs

Reply:
That makes sense. How about using the `Error = errs.Class("console error")` and then `Error.Wrap(err)`. I saw this pattern in other services